mgmtci.proj

```diff
@@ -0,0 +1,12 @@
+<Project DefaultTargets="BuildToolsPackage" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">	
+	<!-- <Import Project="eng/Directory.Build.props" /> -->
+	<Import Project="eng/mgmt/ci.buildtools.props" />
```

Comment:
Is this necessary? Can the projects not be built as a normal traversal projects like the other .NET projects in this repo?

Reply:
@weshaggard my intention was to make things built in the new repo as I had everything setup in our old tooling repo.
If you think things could be made better, can we do this after things are working and the repo is back open.
This way we can start making incremental changes and introduce all the good things that you can suggest.
So to answer to your question, I am open for any good things that can optimize things, but I have one request, let's do it once I get swagger to sdk unblocked.

Reply:
Looks like this isn't needed. We automatically pick-up all csproj files under dotnet directory, see https://github.com/Azure/azure-sdk-tools/blob/master/dotnet.proj#L3. 

Currently CI is failing because you are targeting netcoreapp2.2, change that back to netcoreapp2.1 and then you should they should attempt to build and we will see if there are any issues.

Reply:
If for some reason you want to keep netcoreapp2.2 we can also update the .NET Core sdk we use in this repo.

Reply:
I have changed it to netcoreapp2.1
https://github.com/Azure/azure-sdk-tools/pull/46/files#diff-035dd543be24bf810968eb16330f22faR22
https://github.com/Azure/azure-sdk-tools/pull/46/files#diff-11fa4cbd1212117154d97b9b9ab29997R11